tests/backup/backup_test_labels.go

```diff
@@ -0,0 +1,393 @@
+	BackupCRsThenMultipleRestoresOnHigherK8sVersion                  TestCaseName = "BackupCRsThenMultipleRestoresOnHigherK8sVersion"
+	ScheduleBackupDeleteAndRecreateNS                                TestCaseName = "ScheduleBackupDeleteAndRecreateNS"
+	DeleteNSDeleteClusterRestore                                     TestCaseName = "DeleteNSDeleteClusterRestore"
+	AlternateBackupBetweenNfsAndS3                                   TestCaseName = "AlternateBackupBetweenNfsAndS3"
+	BackupNamespaceInNfsRestoredFromS3                               TestCaseName = "BackupNamespaceInNfsRestoredFromS3"
+	DeleteS3ScheduleAndCreateNfsSchedule                             TestCaseName = "DeleteS3ScheduleAndCreateNfsSchedule"
+	KubeAndPxNamespacesSkipOnAllNSBackup                             TestCaseName = "KubeAndPxNamespacesSkipOnAllNSBackup"
+	MultipleBackupLocationWithSameEndpoint                           TestCaseName = "MultipleBackupLocationWithSameEndpoint"
+	UpgradePxBackup                                                  TestCaseName = "UpgradePxBackupWithHelm"
+	StorkUpgradeWithBackup                                           TestCaseName = "StorkUpgradeWithBackup"
+	PXBackupEndToEndBackupAndRestoreWithUpgrade                      TestCaseName = "PXBackupEndToEndBackupAndRestoreWithUpgrade"
+	IssueDeleteOfIncrementalBackupsAndRestore                        TestCaseName = "IssueDeleteOfIncrementalBackupsAndRestore"
+	DeleteIncrementalBackupsAndRecreateNew                           TestCaseName = "DeleteIncrementalBackupsAndRecreateNew"
+	DeleteBucketVerifyCloudBackupMissing                             TestCaseName = "DeleteBucketVerifyCloudBackupMissing"
+	DeleteBackupAndCheckIfBucketIsEmpty                              TestCaseName = "DeleteBackupAndCheckIfBucketIsEmpty"
+	KubevirtVMBackupRestoreWithDifferentStates                       TestCaseName = "KubevirtVMBackupRestoreWithDifferentStates"
+	BackupAlternatingBetweenLockedAndUnlockedBuckets                 TestCaseName = "BackupAlternatingBetweenLockedAndUnlockedBuckets"
+	LockedBucketResizeOnRestoredVolume                               TestCaseName = "LockedBucketResizeOnRestoredVolume"
+	LockedBucketResizeVolumeOnScheduleBackup                         TestCaseName = "LockedBucketResizeVolumeOnScheduleBackup"
+	DeleteLockedBucketUserObjectsFromAdmin                           TestCaseName = "DeleteLockedBucketUserObjectsFromAdmin"
+	VerifyRBACForInfraAdmin                                          TestCaseName = "VerifyRBACForInfraAdmin"
+	VerifyRBACForPxAdmin                                             TestCaseName = "VerifyRBACForPxAdmin"
+	VerifyRBACForAppAdmin                                            TestCaseName = "VerifyRBACForAppAdmin"
+	VerifyRBACForAppUser                                             TestCaseName = "VerifyRBACForAppUser"
+)
+
+// Test case labels
+const (
+	CreateMultipleUsersAndGroupsLabel                                     TestCaseLabel = "CreateMultipleUsersAndGroups"
```

Comment:
Like Step, I think it is better to create a wrapper function that appends the test name to the labels along with other labels in the map. This Describe could be in backup_helper to avoid affecting other tests.

Reply:
Since Descrube is a container node, I doubt whether we can use a wrapper, but if it works, it would help reduce this redundancy

Reply:
Yes. We can try it out. But this is just an initial PR. Can we take it as an enhancement in future?

Reply:
Sure.

Reply:
Sure.

Reply:
Sure.